portal-web/docroot/WEB-INF/shielded-container-web.xml

```diff
@@ -249,10 +249,6 @@
 		<servlet-name>Combo Servlet</servlet-name>
 		<url-pattern>/combo/*</url-pattern>
 	</servlet-mapping>
-	<servlet-mapping>
```

Comment:
@rafaprax You removed the servlet mapping for "Dynamic Resource Servlet" here which means corresponding servlet can not be accessed, you should also remove the servlet related code.